frontend/public/components/storage-class.tsx

```diff
@@ -36,7 +36,7 @@ const StorageClassTableHeader = () => {
       props: { className: tableColumnClasses[1] },
     },
     {
-      title: <React.Fragment>Reclaim <span className="pf-u-display-none-on-md pf-u-display-inline-block-on-lg">Policy</span></React.Fragment>,
+      title: <React.Fragment>Reclaim <span className="hidden-sm">Policy</span></React.Fragment>,
```

Comment:
Do we need `hidden-xs` as well?

Reply:
No as the col is hidden at xs.

Reply:
`pf-u-display` helpers could be used globally in the future too I suppose... if the concern is having them loaded once (i get this). 

I think more or less this should just show PF4 has the same level of support as BS3.

Reply:
@spadgett and I are both of the opinion we should not be adding PF4 equivalents of things that are already available and in use in the code base.  It's just adding additional code to accomplish the same thing (not to mention the overhead of carrying, maintaining, and debugging duplicate things).

Reply:
The concern is having two ways of doing the same thing, which will get confusing. I'd like to use a single grid system consistency throughout console or we'll go nuts remembering the subtle differences between the two :) Especially if the breakpoint names don't match (`xl` vs `lg`, etc.).

Changing to use a different grid is an enormous change with no visual difference in the end. I feel like it's much greater value to focus on transitions to the PF4 components that update the look & feel of the console to the PF4 look & feel.